src/components/Collections/AddToCollectionModal.tsx

```diff
@@ -380,10 +381,16 @@ function NewCollectionForm({
                 name="mode"
                 label="Mode"
                 data={[
-                  ...Object.values(CollectionMode).map((value) => ({
-                    value,
-                    label: getDisplayName(value),
-                  })),
+                  ...Object.values(CollectionMode)
+                    .map((value) =>
+                      [CollectionMode.Bookmark].some((v) => v === value)
+                        ? undefined
+                        : {
+                            value,
+                            label: getDisplayName(value),
+                          }
+                    )
+                    .filter(isDefined),
```

Comment:
This seems a little bit over engineered 😛 won't this yield the same result?
```suggestion
                  ...Object.values(CollectionMode)
                    .filter((v) => v !== CollectionMode.Bookmark)
                    .map((value) => ({
                      value,
                      label: getDisplayName(value),
                    })),
```

Reply:
Made so that if we need to filter more in the future we can 😎